### ISSUE TITLE
İ forced cloudflare tunnel to use https only , but cloudflare tries to make http

### ISSUE THREAD

**Issue by HafizBeyy** [nextcloud/android#13354]
### Cause of error
```java
Exception in thread "main" kotlin.UninitializedPropertyAccessException: lateinit property currentAccount has not been initialized
    at com.nextcloud.client.mixins.SessionMixin.getCurrentAccount(SessionMixin.kt:32)
    at com.nextcloud.client.mixins.SessionMixin.onRestart(SessionMixin.kt:100)
    at com.nextcloud.client.mixins.MixinRegistry.onRestart(MixinRegistry.kt:48)
    at com.owncloud.android.ui.activity.BaseActivity.onRestart(BaseActivity.java:118)
    at android.app.Instrumentation.callActivityOnRestart(Instrumentation.java:1553)
    at android.app.Activity.performRestart(Activity.java:8445)
    at android.app.ActivityThread.performRestartActivity(ActivityThread.java:5260)
    at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:249)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:204)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:186)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:134)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loopOnce(Looper.java:201)
    at android.os.Looper.loop(Looper.java:288)
    at android.app.ActivityThread.main(ActivityThread.java:7924)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)

```

### App information
* ID: `com.nextcloud.client`
* Version: `30290190`
* Build flavor: `generic`

### Device information
* Brand: `Redmi`
* Device: `curtana`
* Model: `Redmi Note 9S`
* Id: `TQ3A.230901.001`
* Product: `lineage_miatoll`

### Firmware
* SDK: `33`
* Release: `13`
* Incremental: `e17e1b92a3`

**Comment by joshtrichards** [comment]
Duplicate of #13241 
Fixed in #13248 

P.S. The subject line sounds like something to ask about on the Nextcloud Help Forum - https://help.nextcloud.com

**Comment by HafizBeyy** [comment]
> Duplicate of #13241 
> Fixed in #13248 
> 
> P.S. The subject line sounds like something to ask about on the Nextcloud Help Forum - https://help.nextcloud.com

İ had a mistyping. İ mean nextcloud
